engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -24,7 +24,7 @@ sealed trait JobDescriptor[K <: CallKey] {
   def locallyQualifiedInputs: CallInputs
 }
 
-final case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
+case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
```

Comment:
Apparently you can't mock methods brought in from `implicit class`es either, having fun with that... :frowning: